time stamp
uint8 mode
int8 pwm_l
int8 pwm_r
float64 rvel_l
float64 rvel_r
float64 angle_l
float64 angle_r